packages/react-i18n/src/i18n.ts

```diff
@@ -173,9 +174,14 @@ export default class I18n {
     const {style = undefined, ...formatOptions} = options || {};
 
     if (style) {
-      return style === DateStyle.Humanize
-        ? this.humanizeDate(date, formatOptions)
-        : this.formatDate(date, {...formatOptions, ...dateStyle[style]});
+      if (style === DateStyle.HumanizeWithTime) {
```

Comment:
I got rid of humanize with time in web. Humanize covers it all now: https://github.com/Shopify/web/blob/master/packages/@shopify/legacy-i18n/src/Intl.ts#L177